anti.md

```diff
@@ -5,40 +5,78 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Doing laundry
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Doing laundry is such a pain.  No one enjoys doing it.  It takes a lot of work
+sorting everything by material and color, and then actually washing, drying,
+folding and then putting away your clothes is a pain (not to mention things that
+have to be ironed and/or hung up).  This helps people who don't like doing 
+laundry, but not really.  
+
+You can think of how to do laundry as one set of rules that could be globally
+applied to all laundry.  It doesn't take writing a new program to do laundry
+in a different home - just a robot that knows the rules.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A language is appropriate because it lets you be very expressive about what
+sort of things should be washed/dryed/post-processed in what way, and what sort
+of detergents and such, etc.
+
+This isn't appropriate because a single app can handle all of this - just 
+provide a config file.
```

Comment:
Good point. In any case, what I was getting at was the idea of thinking of a config file as a DSL, which I think wouldn't necessarily be unreasonable